test/edit-column-overlay.test.js

```diff
@@ -69,14 +72,15 @@ const fixtures = {
       }
     });
 
-    it('should not stop editing when focusing input related overlay', () => {
+    it('should not stop editing when focusing input related overlay', async () => {
       enter(dateCell);
       const datePicker = getCellEditor(dateCell).querySelector('vaadin-date-picker');
       datePicker.click();
 
-      clickOverlay(datePicker);
+      await clickOverlay(datePicker);
       grid._debouncerStopEdit && grid._debouncerStopEdit.flush();
 
+      await new Promise((resolve) => requestAnimationFrame(resolve));
```

Comment:
```suggestion
      await nextFrame();
```

And also update the import above to look like this:

```js
import { fixtureSync, nextFrame } from '@open-wc/testing-helpers';
```

Reply:
Done.